src/Domain/Donation.php

```diff
@@ -796,7 +796,7 @@ public function getDonorCountryCode(): ?string
      */
     public function getFundingWithdrawalTotal(): string
     {
-        $withdrawalTotal = '0.0';
```

Comment:
adjusted to match what the Money class returns otherwise an assert in production code fails, and I think the format that it would return if non-zero.